lnd.go

```diff
@@ -395,6 +396,10 @@ func Main(cfg *Config, lisCfg ListenerCfg, interceptor signal.Interceptor) error
 	// We'll create the WalletUnlockerService and check whether the wallet
 	// already exists.
 	pwService := createWalletUnlockerService(cfg)
+	if cfg.Cluster.EnableLeaderElection {
```

Comment:
I think we can get rid of this and the two parameters `RemoteDB` and `RemoteWalletDB` in `btcwallet.Config` if we just pass in a `LoaderOptionFunc` to the unlocker service and the `waitForWalletPassword`. That way both the unlocker service as well as the btcwallet package become completely agnostic to the location of the wallet.

Reply:
Yes indeed! Wasn't sure about the amount of refactoring to do, so glad you gave me ideas on how to do this elegantly. PTAL.